### ISSUE TITLE
Limit of 1 message per number

### ISSUE THREAD

**Issue by FabxBs** [FDX100/GOD-KILLER#1]
Is there any way to remove this harassing limit? It makes no sense making an sms tool just to send one message per day.

**Comment by GlitchKJ** [comment]
> Is there any way to remove this harassing limit? It makes no sense making an sms tool just to send one message per day.

changing the timezone worked for me ;)......if you're a debian user
type dpkg-reconfigure tzdata on termux

**Comment by megalopsatlanticus** [comment]
Agree with GlitchKJ...worked for me.

**Comment by Fabio-Git-dev** [comment]
thanks!

**Comment by Fabio-Git-dev** [comment]
I'm on kali linux tho, how do i do it in there?

**Comment by GlitchKJ** [comment]
> I'm on kali linux tho, how do i do it in there?

Bro type dpkg-reconfigure tzdata on termux or use tor

**Comment by Fabio-Git-dev** [comment]
> m

i tried the tzdata, doesn't work, i'm gonna try with tor

**Comment by GlitchKJ** [comment]
> > m
> 
> i tried the tzdata, doesn't work, i'm gonna try with tor, do you have discord?

Try with that....I am currently not active on discord...

**Comment by Fabio-Git-dev** [comment]
> > > m
> > 
> > 
> > i tried the tzdata, doesn't work, i'm gonna try with tor, do you have discord?
> 
> Try with that....I am currently not active on discord...

fact is, i tried tor and the message doesn't send, i try to change timezone, and it says you've sended too many messages for this number

**Comment by GlitchKJ** [comment]
> > > > m
> > > 
> > > 
> > > i tried the tzdata, doesn't work, i'm gonna try with tor, do you have discord?
> > 
> > 
> > Try with that....I am currently not active on discord...
> 
> fact is, i tried tor and the message doesn't send, i try to change timezone, and it says you've sended too many messages for this number

I tried the timezone method last year and it worked for me......maybe you should try again tomorrow or clearing the cache memory may work for you......I was also using Kali Linux but it was in live mode......

**Comment by Fabio-Git-dev** [comment]
live mode?

**Comment by GlitchKJ** [comment]
> live mode?

Yep....You can have Kali Linux installed on your hard drive or you can use a bootable usb for Kali Linux.....I was using a bootable USB for Kali Linux....which clears the memory as well as changes I made to it when I plug out the USB

**Comment by GlitchKJ** [comment]
Btw , You can use any other project for sending custom messages...there are lots of them on GitHub which uses custom api's

**Comment by Fabio-Git-dev** [comment]
i have it on VMWare

**Comment by Fabio-Git-dev** [comment]
if there are other projects that doesn't require an account link them here

**Comment by GlitchKJ** [comment]
> if there are other projects that doesn't require an account link them here

You can always find them yourself! :-!

**Comment by Fabio-Git-dev** [comment]
...

**Comment by GlitchKJ** [comment]
> i have it on VMWare

Version???

**Comment by Fabio-Git-dev** [comment]
16

**Comment by megalopsatlanticus** [comment]
> if there are other projects that doesn't require an account link them here

Just search SMS bomber...there's like 30 of them. TBomb is pretty good. 
git clone https://github.com/TheSpeedX/TBomb.git

I respond with more...but you forgot to say please.

**Comment by Fabio-Git-dev** [comment]
*Please*

**Comment by Fabio-Git-dev** [comment]
By the way would be great if i could send custom messages.
